[2024-11-10T02:03:46.314+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-10T02:03:46.337+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline.insert_data_into_data_warehouse manual__2024-11-10T02:03:43.560664+00:00 [queued]>
[2024-11-10T02:03:46.345+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline.insert_data_into_data_warehouse manual__2024-11-10T02:03:43.560664+00:00 [queued]>
[2024-11-10T02:03:46.345+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-10T02:03:46.357+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): insert_data_into_data_warehouse> on 2024-11-10 02:03:43.560664+00:00
[2024-11-10T02:03:46.361+0000] {standard_task_runner.py:64} INFO - Started process 427 to run task
[2024-11-10T02:03:46.363+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline', 'insert_data_into_data_warehouse', 'manual__2024-11-10T02:03:43.560664+00:00', '--job-id', '458', '--raw', '--subdir', 'DAGS_FOLDER/pipline.py', '--cfg-path', '/tmp/tmpcx1sd_3t']
[2024-11-10T02:03:46.365+0000] {standard_task_runner.py:91} INFO - Job 458: Subtask insert_data_into_data_warehouse
[2024-11-10T02:03:46.404+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline.insert_data_into_data_warehouse manual__2024-11-10T02:03:43.560664+00:00 [running]> on host 8305ceda4491
[2024-11-10T02:03:46.511+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipeline' AIRFLOW_CTX_TASK_ID='insert_data_into_data_warehouse' AIRFLOW_CTX_EXECUTION_DATE='2024-11-10T02:03:43.560664+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-10T02:03:43.560664+00:00'
[2024-11-10T02:03:46.512+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-10T02:03:46.524+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:46.548+0000] {pipline.py:180} ERROR - Error creating or retrieving journal 'IEEE Transactions on Technology and Society': string indices must be integers, not 'str'
[2024-11-10T02:03:46.548+0000] {pipline.py:226} WARNING - Journal 'IEEE Transactions on Technology and Society' was not inserted.
[2024-11-10T02:03:46.553+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:46.562+0000] {pipline.py:180} ERROR - Error creating or retrieving journal 'IEEE Transactions on Technology and Society': string indices must be integers, not 'str'
[2024-11-10T02:03:46.563+0000] {pipline.py:226} WARNING - Journal 'IEEE Transactions on Technology and Society' was not inserted.
[2024-11-10T02:03:46.568+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:46.583+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:46.604+0000] {pipline.py:205} INFO - Publication 'The Generative AI Landscape in Education: Mapping the Terrain of Opportunities, Challenges, and Student Perception' inserted successfully.
[2024-11-10T02:03:46.609+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:46.657+0000] {pipline.py:234} ERROR - Error processing publication 'The Generative AI Landscape in Education: Mapping the Terrain of Opportunities, Challenges, and Student Perception': (1048, "Column 'JournalMain' cannot be null")
[2024-11-10T02:03:46.663+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:46.672+0000] {pipline.py:180} ERROR - Error creating or retrieving journal 'IEEE Computer Architecture Letters': string indices must be integers, not 'str'
[2024-11-10T02:03:46.672+0000] {pipline.py:226} WARNING - Journal 'IEEE Computer Architecture Letters' was not inserted.
[2024-11-10T02:03:46.677+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:46.686+0000] {pipline.py:180} ERROR - Error creating or retrieving journal 'IEEE Embedded Systems Letters': string indices must be integers, not 'str'
[2024-11-10T02:03:46.686+0000] {pipline.py:226} WARNING - Journal 'IEEE Embedded Systems Letters' was not inserted.
[2024-11-10T02:03:46.691+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:46.706+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:46.727+0000] {pipline.py:205} INFO - Publication 'A Survey of Real-Time Strategy Game AI Research and Competition in StarCraft' inserted successfully.
[2024-11-10T02:03:46.732+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:46.764+0000] {pipline.py:234} ERROR - Error processing publication 'A Survey of Real-Time Strategy Game AI Research and Competition in StarCraft': (1048, "Column 'JournalMain' cannot be null")
[2024-11-10T02:03:46.770+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:46.787+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:46.810+0000] {pipline.py:205} INFO - Publication 'Unraveling ML Models of Emotion With NOVA: Multi-Level Explainable AI for Non-Experts' inserted successfully.
[2024-11-10T02:03:46.816+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:46.854+0000] {pipline.py:234} ERROR - Error processing publication 'Unraveling ML Models of Emotion With NOVA: Multi-Level Explainable AI for Non-Experts': (1048, "Column 'JournalMain' cannot be null")
[2024-11-10T02:03:46.860+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:46.878+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:46.900+0000] {pipline.py:205} INFO - Publication 'AIS-PVT: Long-Time AIS Data Assisted Pyramid Vision Transformer for Sea-Land Segmentation in Dual-Polarization SAR Imagery' inserted successfully.
[2024-11-10T02:03:46.906+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:46.951+0000] {pipline.py:234} ERROR - Error processing publication 'AIS-PVT: Long-Time AIS Data Assisted Pyramid Vision Transformer for Sea-Land Segmentation in Dual-Polarization SAR Imagery': (1048, "Column 'JournalMain' cannot be null")
[2024-11-10T02:03:46.956+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:46.972+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:46.992+0000] {pipline.py:205} INFO - Publication 'Explainable AI for Cheating Detection and Churn Prediction in Online Games' inserted successfully.
[2024-11-10T02:03:46.997+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.030+0000] {pipline.py:234} ERROR - Error processing publication 'Explainable AI for Cheating Detection and Churn Prediction in Online Games': (1048, "Column 'JournalMain' cannot be null")
[2024-11-10T02:03:47.036+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.051+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.079+0000] {pipline.py:205} INFO - Publication 'Towards Measuring Fairness in AI: The Casual Conversations Dataset' inserted successfully.
[2024-11-10T02:03:47.086+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.121+0000] {pipline.py:234} ERROR - Error processing publication 'Towards Measuring Fairness in AI: The Casual Conversations Dataset': (1048, "Column 'JournalMain' cannot be null")
[2024-11-10T02:03:47.127+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.141+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.165+0000] {pipline.py:205} INFO - Publication 'Personas for Artificial Intelligence (AI) an Open Source Toolbox' inserted successfully.
[2024-11-10T02:03:47.171+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.219+0000] {pipline.py:234} ERROR - Error processing publication 'Personas for Artificial Intelligence (AI) an Open Source Toolbox': (1048, "Column 'JournalMain' cannot be null")
[2024-11-10T02:03:47.226+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.247+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.269+0000] {pipline.py:205} INFO - Publication 'A Game AI Competition to Foster Collaborative AI Research and Development' inserted successfully.
[2024-11-10T02:03:47.276+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.314+0000] {pipline.py:234} ERROR - Error processing publication 'A Game AI Competition to Foster Collaborative AI Research and Development': (1048, "Column 'JournalMain' cannot be null")
[2024-11-10T02:03:47.322+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.333+0000] {pipline.py:180} ERROR - Error creating or retrieving journal 'IEEE Transactions on Technology and Society': string indices must be integers, not 'str'
[2024-11-10T02:03:47.334+0000] {pipline.py:226} WARNING - Journal 'IEEE Transactions on Technology and Society' was not inserted.
[2024-11-10T02:03:47.339+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.355+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.375+0000] {pipline.py:205} INFO - Publication 'XAI-IoT: An Explainable AI Framework for Enhancing Anomaly Detection in IoT Systems' inserted successfully.
[2024-11-10T02:03:47.380+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.412+0000] {pipline.py:234} ERROR - Error processing publication 'XAI-IoT: An Explainable AI Framework for Enhancing Anomaly Detection in IoT Systems': (1048, "Column 'JournalMain' cannot be null")
[2024-11-10T02:03:47.417+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.434+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.455+0000] {pipline.py:205} INFO - Publication 'Human-Centered AI in Smart Farming: Toward Agriculture 5.0' inserted successfully.
[2024-11-10T02:03:47.461+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.492+0000] {pipline.py:234} ERROR - Error processing publication 'Human-Centered AI in Smart Farming: Toward Agriculture 5.0': (1048, "Column 'JournalMain' cannot be null")
[2024-11-10T02:03:47.497+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.512+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.531+0000] {pipline.py:205} INFO - Publication 'An Area-Efficient Systolic Array Redundancy Architecture for Reliable AI Accelerator' inserted successfully.
[2024-11-10T02:03:47.536+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.569+0000] {pipline.py:234} ERROR - Error processing publication 'An Area-Efficient Systolic Array Redundancy Architecture for Reliable AI Accelerator': (1048, "Column 'JournalMain' cannot be null")
[2024-11-10T02:03:47.575+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.591+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.613+0000] {pipline.py:205} INFO - Publication 'Unleashing the Power of Edge-Cloud Generative AI in Mobile Networks: A Survey of AIGC Services' inserted successfully.
[2024-11-10T02:03:47.618+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.650+0000] {pipline.py:234} ERROR - Error processing publication 'Unleashing the Power of Edge-Cloud Generative AI in Mobile Networks: A Survey of AIGC Services': (1048, "Column 'JournalMain' cannot be null")
[2024-11-10T02:03:47.655+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.669+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.688+0000] {pipline.py:205} INFO - Publication 'Exploring the Effect of Generative AI on Social Sustainability Through Integrating AI Attributes, TPB, and T-EESST: A Deep Learning-Based Hybrid SEM-ANN Approach' inserted successfully.
[2024-11-10T02:03:47.693+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.725+0000] {pipline.py:234} ERROR - Error processing publication 'Exploring the Effect of Generative AI on Social Sustainability Through Integrating AI Attributes, TPB, and T-EESST: A Deep Learning-Based Hybrid SEM-ANN Approach': (1048, "Column 'JournalMain' cannot be null")
[2024-11-10T02:03:47.731+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.745+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.764+0000] {pipline.py:205} INFO - Publication 'The Ethics of AI in Games' inserted successfully.
[2024-11-10T02:03:47.769+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.804+0000] {pipline.py:234} ERROR - Error processing publication 'The Ethics of AI in Games': (1048, "Column 'JournalMain' cannot be null")
[2024-11-10T02:03:47.810+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.825+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.846+0000] {pipline.py:205} INFO - Publication 'E-XAI: Evaluating Black-Box Explainable AI Frameworks for Network Intrusion Detection' inserted successfully.
[2024-11-10T02:03:47.852+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.884+0000] {pipline.py:234} ERROR - Error processing publication 'E-XAI: Evaluating Black-Box Explainable AI Frameworks for Network Intrusion Detection': (1048, "Column 'JournalMain' cannot be null")
[2024-11-10T02:03:47.890+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.905+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.925+0000] {pipline.py:205} INFO - Publication 'Demystifying Defects: Federated Learning and Explainable AI for Semiconductor Fault Detection' inserted successfully.
[2024-11-10T02:03:47.930+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.966+0000] {pipline.py:234} ERROR - Error processing publication 'Demystifying Defects: Federated Learning and Explainable AI for Semiconductor Fault Detection': (1048, "Column 'JournalMain' cannot be null")
[2024-11-10T02:03:47.971+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:47.987+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:48.009+0000] {pipline.py:205} INFO - Publication 'Developing a Transparent Diagnosis Model for Diabetic Retinopathy Using Explainable AI' inserted successfully.
[2024-11-10T02:03:48.016+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:48.064+0000] {pipline.py:234} ERROR - Error processing publication 'Developing a Transparent Diagnosis Model for Diabetic Retinopathy Using Explainable AI': (1048, "Column 'JournalMain' cannot be null")
[2024-11-10T02:03:48.071+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:48.095+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:48.123+0000] {pipline.py:205} INFO - Publication 'AI and Security, From an Information Security and Risk Manager Standpoint' inserted successfully.
[2024-11-10T02:03:48.132+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:48.181+0000] {pipline.py:234} ERROR - Error processing publication 'AI and Security, From an Information Security and Risk Manager Standpoint': (1048, "Column 'JournalMain' cannot be null")
[2024-11-10T02:03:48.187+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:48.209+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:48.234+0000] {pipline.py:205} INFO - Publication 'Empirical Evaluations of Framework for Adaptive Trust Calibration in Human-AI Cooperation' inserted successfully.
[2024-11-10T02:03:48.240+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:48.282+0000] {pipline.py:234} ERROR - Error processing publication 'Empirical Evaluations of Framework for Adaptive Trust Calibration in Human-AI Cooperation': (1048, "Column 'JournalMain' cannot be null")
[2024-11-10T02:03:48.287+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:48.297+0000] {pipline.py:180} ERROR - Error creating or retrieving journal 'IEEE Transactions on Technology and Society': string indices must be integers, not 'str'
[2024-11-10T02:03:48.298+0000] {pipline.py:226} WARNING - Journal 'IEEE Transactions on Technology and Society' was not inserted.
[2024-11-10T02:03:48.303+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-10T02:03:48.313+0000] {pipline.py:180} ERROR - Error creating or retrieving journal 'IEEE Transactions on Technology and Society': string indices must be integers, not 'str'
[2024-11-10T02:03:48.314+0000] {pipline.py:226} WARNING - Journal 'IEEE Transactions on Technology and Society' was not inserted.
[2024-11-10T02:03:48.315+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-10T02:03:48.315+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-10T02:03:48.322+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline, task_id=insert_data_into_data_warehouse, run_id=manual__2024-11-10T02:03:43.560664+00:00, execution_date=20241110T020343, start_date=20241110T020346, end_date=20241110T020348
[2024-11-10T02:03:48.341+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-11-10T02:03:48.362+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-10T02:03:48.369+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
